### DISCUSSION TITLE
Welcome to FsLab Discussions!

### DISCUSSION THREAD

**Discussion by dsyme** [fslaborg/FsLab#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WalternativE** [comment]
Hi y'all! I'm Gregor, I work as a .NET dev in Austria and like to work on Data Science and ML problems in my spare time. I've been using F# for a couple of years now defaulting to it whenever I have to write some data pipeline. I'm always happy to help out whenever I can.

**Comment by kMutagene** [comment]
Hi there 👋 

I'm Kevin, and i am a phd student in the @CSBiology group of @muehlhaus in the University Kaiserslautern, Germany. 

We are using F# almost exclusively for all kinds of bioinformatics and computational related data analysis pipelines, but also develop our own methods in it. An example for that can be seen in our recently published [TMEA framework](https://github.com/CSBiology/TMEA).

I am the maintainer of [BioFSharp](https://github.com/CSBiology/BioFSharp), [Plotly.NET](https://github.com/plotly/Plotly.NET), and [Dash.NET](https://github.com/plotly/Dash.NET). I am also in contact with the guys @plotly, who are very interested in how this discussion is shaping.

I am looking forward to work together with all of you guys to hopefully shape F# as a happy place for data science in .NET!

**Reply by zyzhu** [reply]
Great job to keep @plotly in the loop. It's a win-win situation. For Plotly, it has potential to gain customer awareness in the dotnet enterprise space. For fsharp, Plotly.Net could be a gateway to data science community who never heard of fsharp before.

**Comment by bvenn** [comment]
Hello all,

I'm Benedikt and just like Kevin I am a phd student at @CSBiology of @muehlhaus. 
I'm the maintainer of [FSharp.Stats](https://github.com/fslaborg/FSharp.Stats) which has just been migrated to FsLab:rocket:.

Looking forward to work with you all.

**Comment by zyzhu** [comment]
Hello all. This is Zhenyong Zhu. I work at an investment firm in DC. I started to adopt F# since 2014 after taking a fast-track course by @tpetricek in London. Best decision ever. 

I am the current maintainer of [Deedle](https://github.com/fslaborg/deedle). But I am more of an accidental transitional maintainer who brought Deedle back to live by updating it to netstandard. I did try my best to fix various bugs and added some wrappers to work with MathNet.Numerics via [Deedle.Math](https://www.nuget.org/packages/Deedle.Math/). However, by no means I understand the internals very well.

More help to the boring but crucial data frame area is welcome.

**Comment by Oceania2018** [comment]
Hello everybody, I'm [Haiping Chen](https://github.com/Oceania2018). So exicted that I will work with you fantastic people. And thanks for @dsyme to connect us as a joint team `FsLab` + `SciSharp`. 

Let .NET rejuvenate in the field of machine learning is our original motivation.

For this moment, I'm focused to make [TensorFlow.NET](https://github.com/SciSharp/TensorFlow.NET) more complete and useable in reality.
I'm glad to do anything that help community grow.

**Comment by ZimmerD** [comment]
Hello everybody!

I'm David and also member of the @CSBiology group.
 
I am a maintainer of [BioFSharp,](https://github.com/CSBiology/BioFSharp) [BioFSharp.Mz](https://github.com/CSBiology/BioFSharp.Mz) and [FSharp.Stats](https://github.com/fslaborg/FSharp.Stats). My research is focused on computational proteomics, where I use F# for daily data analysis tasks [(Deedle](https://github.com/fslaborg/Deedle) ❤️) and the development of toolchains facilitating the processing of mass spectrometry based proteomics data. 

I am looking forward to work with you!

**Comment by HLWeil** [comment]
Hello all =)

I'm Lukas and also member of the @CSBiology group. I'm really glad @kMutagene accidentally stumbled upon the discussion at issue #137 because this recent progress really feels like a great step towards an even greater community!

I maintain [FSharp.FGL](https://github.com/CSBiology/FSharp.FGL). My main research interest are regulatory networks, which goes in line with the prior, but I have used many other great F# libraries (also Hooray for Deedle 🎉 ) and I try to contribute where I can.

I'm especially looking forward to a productive and collegial atmosphere not only in the scope of our working group but now also with all of you guys!

**Comment by dsyme** [comment]
@muehlhaus et al, I've asked @tpetricek for the keys for the @fslaborg twitter account, which I can then hand over to you for broadcast announcements.

Remind me if you don't hear back about this 👍 

Best
Don

**Reply by muehlhaus** [reply]
Super. This is a great idea. Thank you.
I think I will need to improve my twitter skills then. :)

Best,
Timo
